leading-teams/peer-level-meeting.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Peer-level meeting
+meta_description: >
+  Nebulab team leads have a weekly discussion in which they share problems and
+  solutions. Read our Playbook to learn how we keep a shared team culture and
+  support each other!
```

Comment:
Sorry, I overlooked this during my first review. Can you make sure to stay [within 160 characters](https://blog.spotibo.com/meta-description-length/)?